tests/saitests/sai_qos_tests.py

```diff
@@ -60,6 +61,41 @@
 ECN_INDEX_IN_HEADER = 53 # Fits the ptf hex_dump_buffer() parse function
 DSCP_INDEX_IN_HEADER = 52 # Fits the ptf hex_dump_buffer() parse function
 
+def get_rx_port(dp, device_number, src_port_id, dst_mac, dst_ip, src_ip):
+    ip_id = 0xBABE
+    tos = (0 << 2) | 1
+    src_port_mac = dp.dataplane.get_mac(device_number, src_port_id)
+    pkt = testutils.simple_ip_packet(pktlen=64,
+                            eth_dst=dst_mac,
```

Comment:
All the arguments should be aligned

Reply:
I'll leave it as it to be consistent with the rest of the file. I do agree arguments should be aligned.